packages/contractkit/src/wrappers/Exchange.ts

```diff
@@ -17,6 +17,11 @@ import {
  * using a Constant Product Market Maker Model
  */
 export class ExchangeWrapper extends BaseWrapper<Exchange> {
```

Comment:
Add [typedoc comments](https://typedoc.org/guides/doccomments/) (to all methods)